### PR TITLE
ENG-4685 add to the operator the capability to select the tomcat image for entando-de-app

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

File: manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml [hunk 1]
```diff
@@ -268,14 +268,16 @@ spec:
                   description: >
                     Specifies the standard server base image that this EntandoApp has been packaged in. Valid values
                     are wildfly (https://github.com/entando/entando-docker-base-images/tree/master/entando-wildfly17-base)
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml [hunk 2]
```diff
@@ -268,14 +268,16 @@ spec:
...
+                    or tomcat (https://github.com/entando/entando-app-engine-base/tree/develop/tomcat)
                     or eap (https://github.com/entando/entando-docker-base-images/tree/master/entando-eap72-clustered-base).
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml [hunk 3]
```diff
@@ -347,14 +347,16 @@ spec:
                   description: >
                     Specifies the standard server base image that this EntandoApp has been packaged in. Valid values
                     are wildfly (https://github.com/entando/entando-docker-base-images/tree/master/entando-wildfly17-base)
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml [hunk 4]
```diff
@@ -347,14 +347,16 @@ spec:
...
+                    or tomcat (https://github.com/entando/entando-app-engine-base/tree/develop/tomcat)
                     or eap (https://github.com/entando/entando-docker-base-images/tree/master/entando-eap72-clustered-base).
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml [hunk 5]
```diff
@@ -347,14 +347,16 @@ spec:
                   description: >
                     Specifies the standard server base image that this EntandoApp has been packaged in. Valid values
                     are wildfly (https://github.com/entando/entando-docker-base-images/tree/master/entando-wildfly17-base)
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml [hunk 6]
```diff
@@ -347,14 +347,16 @@ spec:
...
+                    or tomcat (https://github.com/entando/entando-app-engine-base/tree/develop/tomcat)
                     or eap (https://github.com/entando/entando-docker-base-images/tree/master/entando-eap72-clustered-base).
```
Comment (zonia3000) [inline manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml]: can we change also this link?

File: manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml (hunk 2 above)
Comment (entgigi) [reply manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml]: done

File: manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml (hunk 1 above)
Comment (entgigi) [reply manifests/k8s-116-and-later/cluster-resources/entandoapps.entando.org.crd.yaml]: done

File: manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml (hunk 3 above)
Comment (entgigi) [reply manifests/k8s-116-and-later/cluster-scoped-deployment/all-in-one.yaml]: done

File: manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml (hunk 5 above)
Comment (entgigi) [reply manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml]: done

File: manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml (hunk 6 above)
Comment (entgigi) [reply manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml]: done

**Review by zonia3000 — APPROVED** [review]

**Review by firegloves — APPROVED** [review]
LGTM

**Pull request merged**